test/bin/run-tests-ubuntu.sh

```diff
@@ -36,6 +36,7 @@ clean_up () {
   rm .coverage
   coverage combine
   coverage report -m
+  coverage html
```

Comment:
I assume this generates an HTML report. Could we make this optional? (Env var, flag, etc).
Relates #600 w/r/t keeping test iteration time short.